### PR TITLE
[UI] Interactive Task re-wording

### PR DESCRIPTION
**Pull request by shubhamgupta2956**
Solved #578

### CODE REVIEW

File: ui/docs/tutorial/debug-task.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ VNC connection to a running task execution environment.
 
 There are two ways to access this mode. The easiest is to enter the task's ID
 in the [task inspector](https://tools.taskcluster.net/groups). Under
-"actions", select "create interactive task". If you have permission to create
+"actions", select "Create with SSH task". If you have permission to create
```
Comment (helfi92) [inline ui/docs/tutorial/debug-task.md]: The Interactive connect also allows a user to connect via VNC. Let's rename "SSH" to "SSH/VNC".

File: ui/src/views/Tasks/ViewTask/index.jsx [hunk 2]
```diff
@@ -313,7 +313,7 @@ export default class ViewTask extends Component {
   };
 
   handleCreateInteractiveTaskClick = () => {
-    const title = 'Create Interactive';
+    const title = 'Create with SSH';
```
Comment (helfi92) [inline ui/src/views/Tasks/ViewTask/index.jsx]: "SSH" -> "SSH/VNC"

File: ui/src/views/Tasks/ViewTask/index.jsx [hunk 3]
```diff
@@ -822,7 +822,7 @@ export default class ViewTask extends Component {
                     disabled: actionLoading,
                   }}
                   icon={<ConsoleLineIcon />}
-                  tooltipTitle="Create Interactive"
+                  tooltipTitle="Create with SSH"
```
Comment (helfi92) [inline ui/src/views/Tasks/ViewTask/index.jsx]: "SSH" -> "SSH/VNC"

**Comment by shubhamgupta2956** [comment]
Thanks for review @helfi92 
Please guide me about what should I do next.

**Comment by helfi92** [comment]
My pleasure! What do you think about
* https://github.com/taskcluster/taskcluster/issues/581?
* https://github.com/taskcluster/taskcluster/issues/598

They are really similar. You could start with the first one and if everything is okay, you could help fix the second issue :) If you are interested, please comment directly on the issue so that it becomes assigned to you.

**Pull request merged**